src/components/UploadSection.svelte

```diff
@@ -23,31 +23,34 @@
 
     async function uploadRequest() {
         let uploadedFilesResponse: UploadRequestResponse[];
-        
+
         try {
             uploadedFilesResponse = await getUploadRequest();
         } catch (e) {
             console.error(e);
         }
-        if(!uploadedFiles){
+        if (!uploadedFiles) {
             uploadedFiles = uploadedFilesResponse;
-        }
-        else{
-        for( let i:number = 0; i < uploadedFilesResponse.length; i++){
-            let isNotUploaded:boolean = true;
-            
-            for( let j: number = 0; j < uploadedFiles.length; j++){
-                if(uploadedFilesResponse[i].ObjectKey == uploadedFiles[j].ObjectKey){
-                    uploadedFiles[j].StatusName = uploadedFilesResponse[i].StatusName;
-                    isNotUploaded = false;
+        } else {
+            for (let i: number = 0; i < uploadedFilesResponse.length; i++) {
+                let isNotUploaded: boolean = true;
+
+                for (let j: number = 0; j < uploadedFiles.length; j++) {
+                    if (
+                        uploadedFilesResponse[i].ObjectKey ==
+                        uploadedFiles[j].ObjectKey
+                    ) {
+                        uploadedFiles[j].StatusName =
+                            uploadedFilesResponse[i].StatusName;
+                        isNotUploaded = false;
+                    }
```

Comment:
i dont know why, but i commited it earlier and git think that its another things, so i dont undarstand why is git put it here